server/services/converters.go

```diff
@@ -70,3 +71,39 @@ func indexToOperationIdentifier(index int) *types.OperationIdentifier {
 func timestampInMilliseconds(timestamp int64) int64 {
 	return timestamp * 1000
 }
+
+func utf8ToHex(value string) string {
+	encoded := hex.EncodeToString([]byte(value))
```

Comment:
[comment]
You don't actually check here if the string is utf

Reply:
Renamed to `stringToHex`.